eda/src/autogluon/eda/auto/simple.py

```diff
@@ -143,7 +146,7 @@ def analyze_interaction(
     x: Optional[str], default = None
     y: Optional[str], default = None
     hue: Optional[str], default = None
-    viz_args: Optional[dict], default = None
+    chart_args: Optional[dict], default = None
```

Comment:
what is the intention of the name change here? It is a bit more confusing to me as `fig_args` is for `kwargs to pass into chart figure`

Reply:
`chart_args` is used in all other places
Figure is a wrapper around chart and allows controlling size of the plot area.
![image](https://user-images.githubusercontent.com/10080307/210886351-3b204dfb-6cec-4044-ba57-aedcf70a508c.png)